.github/workflows/deploy_to_s3_prod_temp.yml

```diff
@@ -0,0 +1,48 @@
+# This workflow will build the static files
```

Comment:
Why this workflow?

Reply:
We needed a workflow temporarily to push code to production without merging to release. I've removed it now. We had to change branch protection rules too so that `main` branch deploys to production.

Now I've reverted back branch protection rules to ensure that only `release` branch can deploy to production.